data/json/monster_factions.json

```diff
@@ -123,5 +123,20 @@
         "type" : "MONSTER_FACTION",
         "name" : "spider_cellar",
         "base_faction" : "spider"
+    },{
+        "type" : "MONSTER_FACTION",
+        "name" : "mutant",
+        "by_mood" : ["small_animal", "blob", "utility_bot", "mutant_omnivore", "mutant_predator"]
```

Comment:
You could turn the "mutant" faction into a catch-all base faction for all mutants, like the "insect" faction currently is. Then you wouldn't need to specify
```
"by_mood" : ["mutant_predator", "mutant"]
```

in all mutant sub-types, just the
```
"by_mood" : ["mutant"]
```

in the "mutant" faction. Look how it is done with "insect" - nothing is an "insect", so "insect" can be hostile to "insect". "bee" is a subtype of "insect" that is hostile towards "insect" faction, but still friendly towards "bee".